app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+    def create
```

Comment:
Apply good practices, for example for indentation only 2 blank spaces are used
https://github.com/rubocop/ruby-style-guide

Reply:
Thanks, I will take this repository into account to improve and apply good practices in Ruby.